mindmap/public/js/src/mindmap.js

```diff
@@ -295,9 +298,11 @@ function MindMapXBlock(runtime, element, context) {
       .click(function () {
```

Comment:
I would create a constant called `xblockId` replace all these for `xblockId` instead of having `context.xblock_id`

``` 
const xblockId = context.xblock_id;

const grade = $(element).find(`#grade-input_${xblockId}`).val();
const submission_id = $(element).find(`#submission-id-input_${xblockId}`).val();
const module_id = $(element).find(`#module-id-input_${xblockId`).val();

```

Reply:
Done